bee/src/main/res/values/styles.xml

```diff
@@ -9,8 +10,20 @@
         <item name="viewsWeekendColor">@color/aps_barchart_weekend_deactived</item>
     </style>
 
-    <style name="Apisense.Light" parent="Theme.AppCompat.Light.NoActionBar" >
+    <style name="Apisense.Light" parent="Theme.AppCompat.Light.NoActionBar">
         <item name="colorAccent">@color/aps_accent</item>
         <item name="colorPrimary">@color/aps_primary_dark</item>
     </style>
+
+    <style name="Apisense.ActionBar" parent="android:Widget.Holo.Light.ActionBar.Solid">
+        <item name="android:background">@color/aps_primary</item>
```

Comment:
To remove the rectangle under the cursor, change this line by this one:

`<item name="android:background">@android:color/transparent</item>`

Reply:
This would hide the submenu unfortunately

Reply:
Oups, I hadn't seen that bad behavior.
I retried to find a way to debug this, but without success.